core/src/main/java/org/keycloak/representations/oidc/OIDCClientRepresentation.java

```diff
@@ -125,6 +125,8 @@
     
     private Boolean backchannel_logout_revoke_offline_tokens;
 
+    private String authorization_data_types_supported;
```

Comment:
According to https://datatracker.ietf.org/doc/html/draft-ietf-oauth-rar#section-10,
  `authorization_data_types_supported`
might be  
  `authorization_details_types`

Reply:
Yes you are right, I mistakenly followed the old spec version, I will change.